style.css

```diff
@@ -107,3 +110,357 @@ body {
 .fab {
   color: #505f79;
 }
+
+/* works section starts here */
+
+.works-section-container {
+  background-color: #e5e5e5;
+  width: 100%;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin-top: 114px;
+  display: grid;
+  grid-template-rows: 552px 552px 552px 552px;
+  row-gap: 88px;
+}
+
+.card-container {
+  background-color: #fff;
+  margin: 0 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  border-radius: 16px;
+}
```

Comment:
- [ ] Please add a border style as shown in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). 😁